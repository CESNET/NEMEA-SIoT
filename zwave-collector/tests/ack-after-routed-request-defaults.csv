time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2019-07-29T17:40:27.648,3,fe90821d03810111010010042503003b9140000010
2019-07-29T17:40:41.642,2,fe90821d0353090d012503ff8200
