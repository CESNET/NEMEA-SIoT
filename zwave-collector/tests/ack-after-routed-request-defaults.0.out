3,2019-07-29T17:40:27.648,4270883357,0,3,4,1,0,1,1,17,3,"Switch Binary","",04,250300,"Request","Singlecast"
3,2019-07-29T17:40:41.642,4270883357,1,2,0,1,0,0,9,13,0,"","",,,"","Ack"
